--- libnl-0.5.0/lib/Makefile~	2005-06-19 07:29:28.695202856 +0000
+++ libnl-0.5.0/lib/Makefile	2005-06-19 07:31:27.378160304 +0000
@@ -67,12 +67,12 @@
 	find .deps -iname \*\.d -exec rm -f '{}' \;
 
 install:
-	mkdir -p $(DESTDIR)$(prefix)/lib/
-	install -m 0644 $(OUT_SLIB) $(DESTDIR)$(prefix)/lib
-	rm -f $(DESTDIR)$(prefix)/lib/$(LN1_SLIB) ; \
-	$(LN) -s $(DESTDIR)$(prefix)/lib/$(OUT_SLIB) $(DESTDIR)$(prefix)/lib/$(LN1_SLIB)
-	rm -f $(DESTDIR)$(prefix)/lib/$(LN_SLIB) ; \
-	$(LN) -s $(DESTDIR)$(prefix)/lib/$(LN1_SLIB) $(DESTDIR)$(prefix)/lib/$(LN_SLIB)
+	mkdir -p $(DESTDIR)$(libdir)/
+	install -m 0644 $(OUT_SLIB) $(DESTDIR)$(libdir)
+	rm -f $(LN1_SLIB) ; \
+	$(LN) -sf $(OUT_SLIB) $(DESTDIR)$(libdir)/$(LN1_SLIB)
+	rm -f $(LN_SLIB) ; \
+	$(LN) -sf $(LN1_SLIB) $(DESTDIR)$(libdir)/$(LN_SLIB)
 
 $(DEPS): ../Makefile.opts
 
--- libnl-0.5.0/Makefile.opts.in~	2005-04-04 23:08:04.000000000 +0000
+++ libnl-0.5.0/Makefile.opts.in	2005-06-19 07:33:50.050470816 +0000
@@ -30,6 +30,7 @@
 LIBNL_LIB        := @LIBNL_LIB@
 
 prefix           := @prefix@
+libdir           := @libdir@
 mandir           := @mandir@
 sysconfdir       := @sysconfdir@
 
