--- libnl-1.1/include/netlink/object.h~	2008-01-14 17:48:45.000000000 +0200
+++ libnl-1.1/include/netlink/object.h	2009-05-28 00:47:33.434187052 +0300
@@ -55,7 +55,6 @@
 /* Access Functions */
 extern int			nl_object_get_refcnt(struct nl_object *);
 extern struct nl_cache *	nl_object_get_cache(struct nl_object *);
-extern inline void *		nl_object_priv(struct nl_object *);
 
 #ifdef __cplusplus
 }
