--- libnl-1.0-pre5/lib/Makefile.static	2006-05-26 01:37:40.000000000 -0400
+++ libnl-1.0-pre5/lib/Makefile	2006-05-26 01:38:31.000000000 -0400
@@ -28,6 +28,7 @@
 OBJ      := $(CIN:%.c=%.o)
 CFLAGS   += -fPIC
 OUT_SLIB := $(PACKAGE_NAME).so.$(PACKAGE_VERSION)
+OUT_AR   := $(PACKAGE_NAME).a
 LN_SLIB  := $(PACKAGE_NAME).so
 LN1_SLIB := $(LN_SLIB).1
 
@@ -38,7 +39,8 @@
 
 all:
 	@echo "  MAKE $(OUT_SLIB)"; \
-	$(MAKE) $(OUT_SLIB)
+	$(MAKE) $(OUT_SLIB); \
+	$(MAKE) $(OUT_AR)
 
 $(OUT_SLIB): ../Makefile.opts $(OBJ)
 	@echo "  LD $(OUT_SLIB)"; \
@@ -48,6 +50,10 @@
 	@echo "  LN $(LN1_SLIB) $(LN_SLIB)"; \
 	rm -f $(LN_SLIB) ; $(LN) -s $(LN1_SLIB) $(LN_SLIB)
 
+$(OUT_AR):  ../Makefile.opts $(OBJ)
+	@echo "  AR $(OUT_AR)"; \
+	$(AR) crus $(OUT_AR) $(OBJ)
+
 clean:
 	@echo "  CLEAN lib"; \
 	$(RM) -f $(OBJ) $(OUT_SLIB) $(LN_SLIB) $(LN1_SLIB); \
@@ -59,6 +65,7 @@
 install:
 	mkdir -p $(DESTDIR)$(LIBDIR)/
 	install -m 0644 $(OUT_SLIB) $(DESTDIR)$(LIBDIR)
+	install -m 0644 $(OUT_AR) $(DESTDIR)$(LIBDIR)
 	rm -f $(DESTDIR)$(LIBDIR)/$(LN1_SLIB) ; \
 	$(LN) -s $(OUT_SLIB) $(DESTDIR)$(LIBDIR)/$(LN1_SLIB)
 	rm -f $(DESTDIR)$(LIBDIR)/$(LN_SLIB) ; \
