--- libnl-3.2.5/configure.in.orig	2012-01-12 13:17:07.000000000 +0100
+++ libnl-3.2.5/configure.in	2012-01-14 07:39:05.603088650 +0100
@@ -23,7 +23,7 @@
 	  [libnl_major_version.libnl_minor_version.libnl_micro_version])
 
 m4_define([libnl_lt_current],
-	  [m4_eval(100 * libnl_minor_version + libnl_micro_version - libnl_lt_revision)])
+	  [m4_eval(100 * libnl_minor_version + libnl_micro_version)])
 
 AC_INIT(libnl, [libnl_version], [http://www.infradead.org/~tgr/libnl/])
 AC_CONFIG_HEADERS([lib/defs.h])
