--- libnl-3.2.0/src/lib/Makefile.am.orig	2011-09-13 19:15:01.000000000 +0200
+++ libnl-3.2.0/src/lib/Makefile.am	2011-09-16 18:46:28.999115930 +0200
@@ -2,7 +2,6 @@
 
 AM_CPPFLAGS  = -Wall -I${top_srcdir}/include -I${top_builddir}/include -D_GNU_SOURCE -DPKGLIBDIR=\"$(pkglibdir)\" -DSYSCONFDIR=\"$(sysconfdir)\" -rdynamic
 AM_LDFLAGS = \
-	-L${top_builddir}/lib \
 	-ldl \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
